readthedocs/templates/core/build-overview.md

```diff
@@ -0,0 +1,36 @@
+{% comment %}
+Template used for generating the build overview page that is posted as a comment on the build PR.
+
+Whitespaces and newlines are important in some places like lists and tables,
+make sure to adjust the tags accordingly, as they introduce newlines.
+{% endcomment %}
+## Documentation build overview
+
+> 📚 [{{ project.name }}](https://{{ PRODUCTION_DOMAIN }}{% url "projects_detail" project.slug %}) project | 🛠️ build [#{{ current_version_build.pk }}](https://{{ PRODUCTION_DOMAIN }}{% url "builds_detail" project.slug current_version_build.pk %}) ({{ current_version_build.commit }}) | 🔍 [preview]({{ current_version.get_absolute_url }})
+
+### Files changed
+
+> Comparing with [{{ base_version.verbose_name }}]({{ base_version.get_absolute_url }}) ({{ base_version_build.commit }}). Click on [↩️]({{ base_version.get_absolute_url }}) to see the file in the base version.
+
+{% if diff.files %}
+
+Top 5 files changed:
+
+{% for file in diff.files|slice:5 %}- {{ file.status.emoji }} [{{ file.filename }}]({{ file.url }}) [↩️]({{ file.base_url }})
+{% endfor %}
+
+<details>
+<summary>Show all {{ diff.files|length }} file(s) changed</summary>
+
+> 📝 {{ diff.modified|length }} file(s) modified | ➕ {{ diff.added|length }} file(s) added | ❌ {{ diff.deleted|length }} file(s) deleted
```

Comment:
I would put this modified/added/deleted files outside the summary, so it's viewable without expanding all the file changes.

Reply:
Yea, if it's a summary, I think it's good to put it outside the details.

Reply:
I'm trying to keep the comment very minimal, since the number of all changed files is already displayed when the comment is collapsed, I put the number of each state inside it. Otherwise, we will need to also move the total number of files outside.

Reply:
That works for me as a starting point. We can always adjust as we go.